### PR TITLE
Issue #65: Comply with PHP 7

### PR DESCRIPTION
**Pull request by tim-bezhashvyly**

### CODE REVIEW

**Comment by tim-bezhashvyly** [comment]
Found an issue.

**Comment by Schrank** [comment]
Tests doesn't run locally, because there is no autoloader - what I'm doing wrong?

**Comment by Vinai** [comment]
@schrank configure phpunit to use phpunit.xml.dist
On the command line by using the -c flag, or in PHPStorm by configuring it in the test run config.

File: tests/Integration/Suites/Complexity/TooManyTypedArgumentsSniffTest.php [hunk 1]
```diff
@@ -1,18 +1,21 @@
...
     }
 
     public function testNoWarningsAddedIfNumberOfTypedArgumentsDoesNotExceedAllowed()
     {
-        $code = 'public function processData(Foo $foo, Bar $bar, array $anArray, Baz $baz, $godKnowsWhatTypeIsIt) { }';
+        $code = <<<EOT
+public function processData(Foo \$foo, Bar \$bar, array \$anArray, string \$aString, Baz \$baz, \$godKnowsWhatTypeIsIt)
+{
+}
+EOT;
```
Comment (Schrank) [inline tests/Integration/Suites/Complexity/TooManyTypedArgumentsSniffTest.php]: I like the HEREDOC NOWDOC (whatever it is) syntax here, better readable!

**Comment by Schrank** [comment]
To have it documentend: vendor/bin/phpunit already includes the autoloader, so it is not needed.

File: tests/Integration/Suites/Complexity/TooManyTypedArgumentsSniffTest.php (hunk 1 above)
Comment (tim-bezhashvyly) [inline tests/Integration/Suites/Complexity/TooManyTypedArgumentsSniffTest.php]: He-he. Initial intention was to fit into 120 characters.

**Pull request merged**